### ISSUE TITLE
No Linux UI

### ISSUE THREAD

**Issue by Joe-0237** [Zettlr/Zettlr#4513]
### On GNU/Linux desktops:
**Workflow issues: solved by using the system titlebar:**
- Window Titlebar secondary click does not work, making many actions inaccessible or needlessly difficult for users who rely on that system menu. 
- Window Titlebar has no title.
- Does not respect system setting for which window management titlebar buttons should be displayed e.g.: close window, hide window, maximize/restore window
- Does not respect system theme for window management button icons

**Other:**
- **Accessibility issue:** Does not respect system theme preference or update when the system theme changes.  
- **Polish issue:** Application appears to be _like_ a naive Windows application on GNU/Linux desktops in most components.

App should use the system tittle bar [and _consider_ moving the buttons into the title bar as they are on mac (which makes sense on GNOME but not KDE Plasma or minimal window managers)].

This is important because Linux users who do not already know they want this app will toss it before trying it because it looks to be of poor quality on their desktops.

### Reproducing

Open the app on GNU/Linux

### Zettlr Version

Stable (most recent version)

### Installation Method

None

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Fedora 38

### Additional Information

Tested on GNOME Desktop on Fedora 38

**Comment by exanrauzer** [comment]
Hello, please check latest beta if some glitches are fixed.

**Comment by nathanlesage** [comment]
> Workflow issues: solved by using the system titlebar

So not a bug, but solved.

> Does not respect system light-dark theme preference or update when the system theme changes by default. Users do want their preference respected by default.

Is unsupported on 2.3 due to a limitation in Electron, current Zettlr 3.0 betas support it.

> Application appears to be like a naive Windows application on GNU/Linux desktops in most components.

Yes, because I do not have the time to test and try good combinations of the various Linux environments. I am a developer, not a designer. I am very glad about anybody who improves any styles, especially the Linux ones.

> (which makes sense on GNOME but not KDE Plasma or minimal window managers)

And there you have it: Most "common" window managers appeared to have that layout. I have to use something to make it look good.

***

Bottom line: Closing as nothing here is a bug, and every issue has been addressed in the current betas.
